src/test/java/org/folio/rest/impl/integrationsuite/EholdingsKbCredentialsImplTest.java

```diff
@@ -57,10 +65,23 @@
 import org.folio.rest.jaxrs.model.KbCredentialsPostRequest;
 import org.folio.rest.jaxrs.model.KbCredentialsPutRequest;
 import org.folio.rest.jaxrs.model.LoadStatusNameEnum;
+import org.folio.service.kbcredentials.KbCredentialsService;
 
 @RunWith(VertxUnitRunner.class)
 public class EholdingsKbCredentialsImplTest extends WireMockTestBase {
 
+  @Autowired
+  @Qualifier("nonSecuredCredentialsService")
+  private KbCredentialsService nonSecuredCredentialsService;
+
+  @Autowired
+  private KbCredentialsService securedCredentialsService;
+
+  @Before
+  public void setUp() throws Exception {
```

Comment:
any missing code here? otherwise can be deleted

Reply:
thanks, removed